Sources/Helpers/OASvgHelper.mm

```diff
@@ -15,6 +15,11 @@
 
 @implementation OASvgHelper
 
++ (BOOL)hasMapImageNamed:(NSString *)name
+{
+    return name && ([self mapImagePathFromSvgResource:[name hasPrefix:@"mx_"] ? name : [@"mx_" stringByAppendingString:name]]) != nil;
```

Comment:
Map image cannot have mx_ images. This request should be explained or completely reworked.